frontend/src/component/feature/FeatureStrategy/FeatureStrategyForm/NewFeatureStrategyForm.tsx

```diff
@@ -373,6 +374,47 @@ export const NewFeatureStrategyForm = ({
                     </StyledEnvironmentBox>
                 ) : null}
             </StyledHeaderBox>
+
+            <ConditionallyRender
+                condition={hasChangeRequestInReviewForEnvironment}
+                show={alert}
+                elseShow={
+                    <ConditionallyRender
+                        condition={Boolean(isChangeRequest)}
```

Comment:
why isChangeRequest not a boolean?

Reply:
FeatureStrategyForm originally accepts it as boolean | undefined for some reason.

Reply:
Looking at the code, it doesn't make sense to have it optional. Fixing it.